sql/16.0/extra/float8.sql

```diff
@@ -1134,6 +1134,9 @@ INSERT INTO "type_FLOAT_INF" (i, f) VALUES (13, 'Inf');
 INSERT INTO "type_FLOAT_INF" (i, f) VALUES (14, '+Inf');
 --Testcase 288:
 INSERT INTO "type_FLOAT_INF" (i, f) VALUES (15, '-Inf');
+--Testcase 289: Inserted as NULL!!!
```

Comment:
Could you update this comment to explain clearly. If someone else reads it, they won't understand why the comment is insert NULL but the testcase is INSERT NaN

Reply:
Changed to `--Testcase 289: SQLite ignores NaN`.

Reply:
Thank you for supporting. It's OK